src/defaultProps.js

```diff
@@ -32,11 +32,11 @@ export default {
   },
   defaultSortMethod: (a, b) => {
     // force null and undefined to the bottom
-    a = (a === null || a === undefined) ? -Infinity : a
-    b = (b === null || b === undefined) ? -Infinity : b
+    a = (a === null || a === undefined) ? '' : a
+    b = (b === null || b === undefined) ? '' : b
     // force any string values to lowercase
-    a = a === 'string' ? a.toLowerCase() : a
-    b = b === 'string' ? b.toLowerCase() : b
+    a = typeof a === 'string' ? a.toLowerCase() : a
+    b = typeof b === 'string' ? b.toLowerCase() : b
```

Comment:
Wow, I wonder if this was a regression that I made on accident.  Very obvious failure on my part.